web/docs/usage/collect-metrics.md

```diff
@@ -37,8 +37,8 @@ e.g., write aggregations over metrics to compute runtime statistics suitable for
 reporting or dashboarding:
 
 ```tql
-metrics
-where @name == "tenzir.metrics.operator" and sink == true
+metrics "operator"
+where output == true
```

Comment:
It's dangerous to have this change in here already if this PR does not reflect that already. Instead, this part of the documentation should be updated when we actually change the output of `metrics "operator"`.